expat/configure.ac

```diff
@@ -231,6 +231,15 @@ AS_IF([test "x${enable_xml_attr_info}" = "xyes"],
       [AC_DEFINE([XML_ATTR_INFO], 1,
          [Define to allow retrieving the byte offsets for attribute names and values.])])
 
+AC_ARG_ENABLE([auto-detect-getrandom],
+  [AS_HELP_STRING([--enable-auto-detect-getrandom],
+                 [Enable auto-detection of getrandom function from include <sys/random.h>
+                  @<:@default=no@:>@])], ,
+                 [enable_auto_detect_getrandom=no])
+AS_IF([test "x${enable_auto_detect_getrandom}" = "xyes"],
+      [AC_DEFINE([HAVE_GETRANDOM], 1,
+      [Define to allow auto-detection of getrandom function from include <sys/random.h>])])
+
```

Comment:
This code seems to ignore rather than integrate with the existing getrandom-detection code in `configure.ac` at https://github.com/libexpat/libexpat/blob/4eeb9af97a3003901df50c7c991613ae27d6ec2c/expat/configure.ac#L179-L203.

What I believe we need for Autotools is two new options, `--without-getrandom` and `--without-sys-getrandom`. By default, auto-detections should remain to be done.